config/default/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+namespace: aws-provider-system
+namePrefix: aws-provider-
+
+resources:
+- ../crds/awsprovider_v1alpha1_awsclusterproviderconfig.yaml
+- ../crds/awsprovider_v1alpha1_awsclusterproviderstatus.yaml
+- ../crds/awsprovider_v1alpha1_awsmachineproviderconfig.yaml
+- ../crds/awsprovider_v1alpha1_awsmachineproviderstatus.yaml
+- ../rbac/rbac_role.yaml
+- ../rbac/rbac_role_binding.yaml
+- ../manager/manager.yaml
+
+patches:
+- ../../cmd/clusterctl/examples/aws/out/aws_manager_image_patch.yaml
+
+secretGenerator:
+- name: machine-controller-credential
+  type: Opaque
+  commands:
+    credentials: "cat ../../cmd/clusterctl/examples/aws/out/credentials"
+
+configMapGenerator:
+- name: machine-setup
+  files:
+  - ../../cmd/clusterctl/examples/aws/out/machines.yaml
```

Comment:
Is this creating a configMap of the machines.yaml? Or am I missing something?

Reply:
That's correct

Reply:
I don't think we have a use case for storing the machines as a config map, the machines should be applied directly from the machines.yaml on the command line from clusterctl.